games-util/lutris/lutris-0.5.0.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
why not 7?

Reply:
With 7 show me that EAPI=7 is not supported on my system

Reply:
then your system is totally obsolete and this needs major updates ;)

Reply:
funtoo user ;)